build.xml

```diff
@@ -38,6 +38,12 @@
     </fileset>
   </path>
 
+  <path id="xmlunit.classpath">
```

Comment:
Does this need to be a separate classpath, or can it just be part of junit.classpath? It looks like they are paired everywhere.

Reply:
Since it was named "junit.classpath" and not, for example, "test.classpath", I thought there might be some reason to keep things separate. No?

Reply:
It was just a change from junit.jar to junit.classpath when a newer JUnit had a second JAR file. Combining them as test.classpath SGTM.